apps/src/templates/AgeDialog.jsx

```diff
@@ -48,7 +49,8 @@ class AgeDialog extends Component {
     // If the song filter override has been turned on, set session storage
     // Dialog won't render
     if (queryString.parse(window.location.search).songfilter === 'on') {
-      this.setManualFilter();
+      this.props.storage.setItem(SONG_FILTER_SESSION_KEY, true);
```

Comment:
Do we actually want this choice to remain in place for the session?  What if the user wants to turn the filter override off again?  I thought that relying on the URL parameter for each page load could be more appropriate.

Reply:
I lean towards making it hard to turn the filter off. When the student goes through the "run, success, continue" path and is navigated to the next page, the url parameter is dropped. Storing the filter in the session cookie will ensure that if a teacher wants to enforce the filter in their classroom and directs students to "studio.code.org/s/dance/lessons/1/levels/1?songfilter=on" and ask them to play through the tutorial, the filter will remain on for tutorial. I believe the '?songfilter=off' part was cut from the [original spec](https://docs.google.com/document/d/1GXo6xekcqjIQ-UGjxcGhfKxZexxxYofck8C_MLUppVM/edit#) so that students wouldn't have an easy time of turning the filter off if they've been directed to turn it on.

Reply:
I see.  If we are changing the existing behavior it seems worth capturing that in the PR description.  I'm a bit concerned about subsequent students on shared computers getting confused, but I guess we use session storage for the signed-out age selector so this is at least consistent.

Reply:
I don't believe the original behavior has changed in regards to whether the filter override lives in a session. Previously, in `setManualFilter`, we were storing the override in the `AGE_DIALOG_SESSION_KEY`. This is also where we were storing whether a signed out user was over/under 13. This PR splits that behavior into two session keys, so we can more accurately differentiate between signedIn with/out override and signedOut with/out override.